plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -163,6 +163,31 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
     cameraAnimationsFactory = CameraAnimatorsFactory(mapDelegateProvider)
   }
 
+  override fun onCameraMove(
+    lat: Double,
+    lon: Double,
+    zoom: Double,
+    pitch: Double,
+    bearing: Double,
+    padding: Array<Double>
+  ) {
+    this.bearing = bearing
+    this.center = Point.fromLngLat(lon, lat)
+    // Insets array order : insets.left, insets.top, insets.right, insets.bottom
+    this.padding = EdgeInsets(
+      /* top = */
+      padding[1],
+      /* left = */
+      padding[0],
+      /* bottom = */
+      padding[3],
+      /* right = */
+      padding[2],
```

Comment:
It's how they're placed in the array, https://github.com/mapbox/mapbox-maps-android/blob/main/sdk/src/main/java/com/mapbox/maps/plugin/MapPluginRegistry.kt#L160

Reply:
yes, I mean why we have the array in that order 🙈 , and probably it's easier to reuse the EdgeInsects class instead of the array there, as it's more clear what the fields are.

Reply:
Well it's part of the public interface so will be problematic to replace array with the Insets. However, the only children of MapCameraPlugin are CompassViewPlugin and this new CameraAnimationsPluginImpl. CompassViewPlugin does not make use of paddings at all, it just uses bearing, https://github.com/mapbox/mapbox-maps-android/blob/main/plugin-compass/src/main/java/com/mapbox/maps/plugin/compass/CompassViewPlugin.kt#L180, so I'll change the order of this array elements to correspond to EdgeInsets.

Reply:
Updated

Reply:
Added more suggestions in https://github.com/mapbox/mapbox-maps-android/pull/1909#discussion_r1050563928

Reply:
Reverted, thanks for pushing

Reply:
FWIW, it seems to follow the same order as in https://developer.android.com/reference/android/graphics/Rect#Rect(int,%20int,%20int,%20int)